index.html

```diff
@@ -16,29 +16,36 @@
 
 <body>
     <header>
-        <nav class="nav">
-        <label class="logo">Shuja</label>
-        <ul>
-            <li><a class="em" href="#">Home</a></li>
-            <li><a class="em" href="#seeee">Portfolio</a></li>
-            <li><a class="em" href="#about">Contact</a></li>
-            <li><a class="em" href="gmail.com"><img src="assets/navicn.png" alt="Email"></a></li>
-            <li><a class="hidedk" href="#"><img src="Union.png" alt="Menu"></a></li>
-        </ul>
+        <nav class="nav" id="nav-bar">
+            <label class="logo">Shuja</label>
+            <a class="hidedk" id="hambuger-menu" href="#"><img src="Union.png" alt="Menu"></a>
+            <svg fill="#FFFFFF" id="btn-close" class="btn-close" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50"
+                width="25px" height="25px">
+                <path
+                    d="M 11.5 11 C 11.372 11 11.243984 11.048984 11.146484 11.146484 C 11.049484 11.244484 11 11.372 11 11.5 C 11 11.628 11.048484 11.755516 11.146484 11.853516 L 24.292969 25 L 11.146484 38.146484 C 10.951484 38.341484 10.951484 38.658516 11.146484 38.853516 C 11.244484 38.950516 11.372 39 11.5 39 C 11.628 39 11.755516 38.951516 11.853516 38.853516 L 25 25.707031 L 38.146484 38.853516 C 38.341484 39.048516 38.658516 39.048516 38.853516 38.853516 C 39.048516 38.657516 39.049516 38.342484 38.853516 38.146484 L 25.707031 25 L 38.853516 11.853516 C 39.048516 11.658516 39.048516 11.341484 38.853516 11.146484 C 38.657516 10.951484 38.342484 10.950484 38.146484 11.146484 L 25 24.292969 L 11.853516 11.146484 C 11.756016 11.048984 11.628 11 11.5 11 z" />
+            </svg>
```

Comment:
- [ ] Kindly give the close and open menu icon some space in right to be like the Figma design and update the font color like the Figma.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/179755548-32d250ae-9a89-4b16-a5e8-183e7c112854.png) |![image](https://user-images.githubusercontent.com/53402600/179755595-83c8f79f-5ea4-4cac-8711-8dc0a9968293.png) |

Reply:
### Hello, Thank you for reviewing my project, Your requested change has been implemented.